functions.php

```diff
@@ -32,13 +32,13 @@ function populateTable(array $vinylDetails): string {
             isset($Details['album']) &&
             isset($Details['year'])) {
 
-            if (isset($Details['cover_art'])) {
```

Comment:
just a thought was your isset not working because it needed to be !isset (so checking it returns false)

Reply:
nope, it was to do with the fact that once you upload a file it becomes an array, even if it's an empty array.. so it is always set, even if it's empty.